docs/Doxyfile.in

```diff
@@ -1099,6 +1099,44 @@ USE_HTAGS              = NO
 
 VERBATIM_HEADERS       = YES
```

Comment:
Git rid of this file in your PR. The Doxyfile.in on develop now matched the doxygen version of ubuntu-latest on the GitHub CI actions. So you should not need to change it.